Microsoft.Dynamics365.UIAutomation.Browser/BrowserOptions.cs

```diff
@@ -152,7 +152,8 @@ public virtual EdgeOptions ToEdge()
         {
             var options = new EdgeOptions()
             {
-                PageLoadStrategy = PageLoadStrategy.Normal
+                PageLoadStrategy = PageLoadStrategy.Normal,
+                UseInPrivateBrowsing = PrivateMode
```

Comment:
see Issue #733 this  [commet](https://github.com/microsoft/EasyRepro/issues/733#issuecomment-587234144)